### PR TITLE
构建文件，及添加测试例子。

### PR DESCRIPTION
**Pull request by 52cik**
.detach() 测试暂时注释了，因为之前pr时这个测试失败了。  
添加了 .getUrlParam()  测试用例。

### CODE REVIEW

**Comment by jaywcjlove** [comment]
不用注视你拉新的依赖包，就不报错了

**Comment by jaywcjlove** [comment]
你pull 代码 dependencies 我更新了版本
```
{
  "dependencies": {
    "chai": "~3.4.1",
    "cli-color": "~1.1.0",
    "gulp": "~3.9.1",
    "mocha": "~2.3.4",
    "mocha-phantomjs": "~4.0.2"
  }
}
```

**Comment by 52cik** [comment]
好的。
